firmware/shared/controls/brake_pedal_lights.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <algorithm>
+#include <iostream>
+
+namespace ctrl {
+
+template <typename T>
+T CalculateBrakePedalPosition(bool rear_brake_pedal_error,
+                              bool front_brake_pedal_error,
+                              T rear_brake_pedal_position,
+                              T front_brake_pedal_position) {
+    // Brake Error Handling
+    if (rear_brake_pedal_error || front_brake_pedal_error) {
+        return T{};  // removed the initalized variable because i realized i can
```

Comment:
This is a bad way to initialize a variable to 0. In not even sure if it's guaranteed to work by the c++ standard. Use `static_cast<T>(0)` instead